go/vt/vttablet/tabletserver/schema/historian.go

```diff
@@ -178,6 +182,13 @@ func (h *historian) loadFromDB(ctx context.Context) error {
 		h.lastID = id
 	}
 	h.sortSchemas()
+
+	loadedSchemas := len(h.schemas)
```

Comment:
loadFromDB is called every time schema changes and the number of entries in `h.schemas` can grow pretty fast